tests/test_frame.py

```diff
@@ -2097,3 +2097,14 @@ def test_groupby_and_transform() -> None:
     check(assert_type(c2, pd.DataFrame), pd.DataFrame)
     check(assert_type(c3, pd.Series), pd.Series)
     check(assert_type(c4, pd.Series), pd.Series)
+
+
+def test_getattr_and_dataframe_groupby() -> None:
+    df = pd.DataFrame(
+        {
+            "col1": [1, 5, 5, 2, 5, 5],
+            "col2": [2.0, 5.0, 8.0, 1.0, 2.0, 9.0],
+            "col3": [9,8,7,5,6,1]
+        }
+    )
+    check(assert_type(df.groupby("col1").__getattr__("col3"), pd.Series), pd.Series)
```

Comment:
This test should be the same as https://github.com/pandas-dev/pandas-stubs/pull/457#discussion_r1039977959

Reply:
Done